docs/pages/style-spec.js

```diff
@@ -539,11 +481,16 @@ export default class extends React.Component {
                                         android: '4.2.0',
                                         ios: '3.4.0',
                                         macos: '0.3.0'
+                                    },
+                                    'line distance metrics': {
```

Comment:
Where is this coming from @katydecorah? Was this addition intentional?

Reply:
Yes, I pulled it from the duplicated content and checked the git history to make sure `line distance metrics` should be included here.